src/ripple/protocol/Rate.h

```diff
@@ -40,10 +41,10 @@ struct Rate
 
     Rate () = default;
 
+    explicit
     Rate (std::uint32_t rate)
```

Comment:
I'd like to zero initialize rate and use this as the default constructor, especially since this class is replacing `STAmount` in many places (which is zero initialized).